json-to-go.js

```diff
@@ -16,7 +16,7 @@ function jsonToGo(json, typename)
 
 	try
 	{
-		data = JSON.parse(json);
+		data = JSON.parse(json.replace(/\.0/g, ".1"));
```

Comment:
:) Yeah, kind of an issue with JS -> Go because numbers are all just numbers in JS. Maybe there's a better solution, but it's all I could come up with right now.